pkgs/development/python-modules/fs/default.nix

```diff
@@ -32,7 +33,7 @@ buildPythonPackage rec {
   # strong cycle with paramaterized
   doCheck = false;
   checkInputs = [ pyftpdlib mock psutil pytestCheckHook ];
-  propagatedBuildInputs = [ six appdirs pytz ]
+  propagatedBuildInputs = [ six appdirs pytz setuptools ]
```

Comment:
That looks dubious, why?

Reply:
But it is correct https://github.com/PyFilesystem/pyfilesystem2/blob/master/fs/__init__.py#L4

Reply:
when I remove "setuptools" then fontmake writes 
```
[max@nixos:~/Документы/nixpkgs]$ fontmake
Traceback (most recent call last):
  File "/nix/store/lfw6mp25fi6yk1h473z8kj9cn42wjmfg-python3.9-fontmake-3.3.0/bin/.fontmake-wrapped", line 6, in <module>
    from fontmake.__main__ import main
  File "/nix/store/lfw6mp25fi6yk1h473z8kj9cn42wjmfg-python3.9-fontmake-3.3.0/lib/python3.9/site-packages/fontmake/__main__.py", line 21, in <module>
    from ufo2ft import CFFOptimization
  File "/nix/store/m80salh8x6jrrjxl59i5m41c8mjl8734-python3.9-ufo2ft-2.27.0/lib/python3.9/site-packages/ufo2ft/__init__.py", line 14, in <module>
    from ufo2ft.outlineCompiler import OutlineOTFCompiler, OutlineTTFCompiler
  File "/nix/store/m80salh8x6jrrjxl59i5m41c8mjl8734-python3.9-ufo2ft-2.27.0/lib/python3.9/site-packages/ufo2ft/outlineCompiler.py", line 36, in <module>
    from ufo2ft.fontInfoData import (
  File "/nix/store/m80salh8x6jrrjxl59i5m41c8mjl8734-python3.9-ufo2ft-2.27.0/lib/python3.9/site-packages/ufo2ft/fontInfoData.py", line 23, in <module>
    from fontTools import ufoLib
  File "/nix/store/fmdwl461hfhj416n905qml8zhjnk3pfi-python3.9-fonttools-4.33.3/lib/python3.9/site-packages/fontTools/ufoLib/__init__.py", line 8, in <module>
    import fs
  File "/nix/store/1q6as9qzraaazq06f8x47igmsyn9dmpl-python3.9-fs-2.4.15/lib/python3.9/site-packages/fs/__init__.py", line 4, in <module>
    __import__("pkg_resources").declare_namespace(__name__)  # type: ignore
ModuleNotFoundError: No module named 'pkg_resources'
```